sections/semantics-links.include

```diff
@@ -1149,7 +1149,7 @@
     <tr>
       <td><{link/stylesheet}></td>
       <td><a>External Resource</a></td>
-      <td><em>not allowed</em></td>
+      <td class="yes"> Yes </td>
```

Comment:
This should have replaced the next line - the one it did replace should still be there because stylesheets are not allowed as `rel` values for `a` and `area` elements.